### PR TITLE
adding option to provide separate url for API

### PR DESCRIPTION
**Pull request by wronan**
Relates to https://github.com/piwik/piwik-log-analytics/issues/128

### CODE REVIEW

File: import_logs.py [hunk 1]
```diff
@@ -837,7 +842,14 @@ def _parse_args(self, option_parser):
 
         if not (self.options.piwik_url.startswith('http://') or self.options.piwik_url.startswith('https://')):
             self.options.piwik_url = 'http://' + self.options.piwik_url
-        logging.debug('Piwik URL is: %s', self.options.piwik_url)
+        logging.debug('Piwik tracker URL is: %s', self.options.piwik_url)
```
Comment (mattab) [inline import_logs.py]: Suggested to change to `Piwik Tracker API URL is` for consistency

File: import_logs.py [hunk 2]
```diff
@@ -837,7 +842,14 @@ def _parse_args(self, option_parser):
...
+
+        if not self.options.piwik_api_url:
+            self.options.piwik_api_url = self.options.piwik_url
+
+        if not (self.options.piwik_api_url.startswith('http://') or self.options.piwik_api_url.startswith('https://')):
+            self.options.piwik_api_url = 'http://' + self.options.piwik_api_url
+        logging.debug('Piwik API URL is: %s', self.options.piwik_api_url)
```
Comment (mattab) [inline import_logs.py]: Suggested to change to `Piwik Analytics API URL is` for consistency

**Comment by mattab** [comment]
@wronan 

Thanks for the PR. Here is my review:
* the new `piwik_api_url` should be used in all places where API is called. we must add it to https://github.com/piwik/piwik-log-analytics/blob/master/import_logs.py#L883

**Comment by mattab** [comment]
@wronan I wanted to merge for 2.16.1 but noticed you have not yet updated the PR to fix the issue that i found during the review. could you apply my review above so it can be merged in 2.16.1?

**Comment by wronan** [comment]
I'll do that after work (in about 10hrs).

**Comment by mattab** [comment]
@wronan Ok let me know if this can be merged for 2.16.1 or we can do it later otherwise

**Comment by mattab** [comment]
ping @wronan

**Comment by wronan** [comment]
Hi @mattab, I pushed post-review changes. Sorry it took so long... there is no excuse for me :( But on the other hand, still better then never (I suppose :D )

**Comment by mattab** [comment]
:+1: nice one

**Pull request merged**